README.md

```diff
@@ -91,9 +91,9 @@ logs are, well, errors.  If your code receives an `error` from a subordinate
 function call and is logging that `error` *and not returning it*, use error
 logs.
 
-2) Verbosity-levels on info logs.  This gives developers a chance to indicate
+2. Verbosity-levels on info logs.  This gives developers a chance to indicate
 arbitrary grades of importance for info logs, without assigning names with
-semantic meaning such as "warning", "trace", and "debug".  Superficially this
+semantic meaning such as "warning", "trace", and "debug."  Superficially this
```

Comment:
Style guides say the period belongs inside the quotes for Americans, but that's silly.

Period outside is better in these technical contexts, I believe.

Reply:
While _you_ might find it silly, from my experience, most technical writing uses the American punctuation rules.